resources/js/Pages/Dashboard/Timesheet/Index.vue

```diff
@@ -137,7 +137,7 @@
                         :label="$t('dashboard.timesheet_create_choose_project')"
                         :data-cy="'project-selector'"
                         :required="true"
-                        @input="showTasks($event)"
+                        @update:modelValue="showTasks($event)"
```

Comment:
This feature is broken.

Reply:
Actually the entire Timesheet view on the dashboard is broken.

Reply:
Fixed!